.changeset/spicy-flowers-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+fix: Exponentially backoff when retrying a call to /upload
```

Comment:
This could be re-written to a be a bit more user-facing. They don't care about the API we're hitting. They care that we're making changes to `wrangler pages publish` and that it should make it more reliable in the event of failures.

Reply:
This is a very good call out! Updating this now.